Dockerfile

```diff
@@ -1,7 +1,15 @@
 FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
-ENV TZ=America/New_York
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN apt-get update \
+      && apt-get upgrade -y \
+      && rm -rf /var/lib/apt/lists/*
+
+ARG TZ
+ENV TZ=${TZ:-America/New_York}
```

Comment:
We can actually remove the TZ environment variable since it's only used for the build. I might do this in another MR.